R/addin.R

```diff
@@ -78,7 +78,16 @@ pkg_search_addin <- function(
     shiny::htmlOutput(paste0("results-", id))
   }
 
-  ui <- shiny::navbarPage("Package search",
+  ui <- shiny::navbarPage(
+    title = div(
+      div(
+        id = "rhub-logo",
+        img(src = "https://cdn.jsdelivr.net/gh/r-hub/branding@master/logo/rhub-square.svg",
```

Comment:
Could this be clickable i.e. with a link to docs.r-hub.io?

Reply:
I mean in the addin could one click on the logo to find what that logo represents?

Reply:
Oh, right.

Reply:
Thanks!